src/compiler/tsc.ts

```diff
@@ -419,7 +417,7 @@ module ts {
     }
 
     function printVersion() {
-        sys.write(getDiagnosticText(Diagnostics.Version_0, version) + sys.newLine);
+        sys.write(getDiagnosticText(Diagnostics.Version_0, ts.version) + sys.newLine);

```

Comment:
I don't NEED it, I just LIKE it :smile:  

The main reason however, is that 'version' is a really common identifier throughout our code base.  I just wanted to make it clear this was the top-level module property, and not some local or closure reference.